monolith/templates/mywall.html

```diff
@@ -9,7 +9,7 @@ <h2>Your Stories</h2>
     <ul>
         <li>
             <h3 style="display: inline;"> <a href="/story/{{story.id}}">{{story.title}}</a></h3>
-            <p style="display: inline;">Posted on ({{story.date}})</p>
+            <p style="display: inline;">Posted on ({{story.date.date()}})</p>
```

Comment:
Little tip: use this instead `({{story.date.strftime("%d/%m/%Y %H:%M")}})`
In this way we have date (correctly formatted) and minutes.